packages/flutter/test/material/text_field_test.dart

```diff
@@ -112,6 +112,7 @@ void main() {
         expect(textFieldValue, equals(testValue));
 
         await tester.pumpWidget(builder());
+        await tester.pump(const Duration(milliseconds: 200));
```

Comment:
if it's not too much of a pain, can you add "// skip past the scrolling animation" or some such to these? it'll make it easier to figure out why we're pumping randomly when debugging these tests in the future.